### PR TITLE
CHORE: switch integration tests to RITEway

### PR DESCRIPTION
**Pull request by krobi64**
[PR Process](https://github.com/poetapp/documentation/blob/master/process/pr-review.md#pr-process) - [PR Review Checklist](https://github.com/poetapp/documentation/blob/master/process/pr-review.md#pr-review-checklist)

### Release
 Semantic release is enabled for this repository. Make sure you follow the right commit message convention. 
 We're using semantic-release's default — [Angular Commit Message Conventions](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines).

## Description of Changes

Moved all of the Alsatian Integration tests to use `riteway` and moved them into `tests/integration/API/WorksController.test.ts`

Please verify that I maintained the same coverage.

### CODE REVIEW

**Comment by warrenv** [comment]
This is a great step! I'm finding the PR too large to make reviewing practical however.
Let's discuss some ways to break this down into smaller steps. I think we can streamline some
parts also (I'm not sure the blockchain integration is needed in the tests, for instance).

File: tests/integration/API/WorkController.test.ts [hunk 1]
```diff
@@ -0,0 +1,488 @@
...
+  await ensureBitcoinBalance(bitcoindClient)
+
+  // Allow everything to finish starting.
+  await delay(5 * 1000)
+
+  // Submit a claim
+  await setUpExistingClaims([TheRaven])
+
+  // Wait for a claim batch to be submitted to the blockchain.
+  await delay(parseInt(process.env.TIMESTAMP_INTERVAL_IN_SECONDS || '10', 10) * 1000 * 2)
```
Comment (ericelliott) [inline tests/integration/API/WorkController.test.ts]: All these big time interval expressions should be made constants so you don't have to repeat the expression copypasta style.

File: tests/integration/API/WorkController.test.ts [hunk 2]
```diff
@@ -0,0 +1,488 @@
...
+  await delay(parseInt(process.env.READ_DIRECTORY_INTERVAL_IN_SECONDS || '5', 10) * 1000 * 2)
+
+  {
+    let theRavenClaim
+    let theRavenResponse
+    try {
+      theRavenResponse = await getWorkFromNode(TheRaven.id)
+      theRavenClaim = await theRavenResponse.json()
+    } catch (err) {
+      theRavenClaim = {}
```
Comment (ericelliott) [inline tests/integration/API/WorkController.test.ts]: Would this provide a better failure report if you set this to the error so you can see it in the `actual` vs `expected` report?

**Comment by krobi64** [comment]
Closing in favor of much smaller PRs:
* #587 
* #590 
*

**Pull request closed without merging**